core/script.runtime/src/kotlin/script/extensions/samWithReceiver.kt

```diff
@@ -16,6 +16,11 @@
 
 package kotlin.script.extensions
 
+// discuss
+
+// Is this an appropriate place to put this class?
```

Comment:
The problem is that now the script template is the only configuration transfer channel for gradle kotlin DSL, therefore the annotation `SamWithReceiverAnnotations` is a natural way to convey the info. If we'll drop it, we'll need to introduce another channel, which is quite unfortunate.